tensorflow/contrib/tensorrt/python/quantization_hints.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Exposes the scope for NVIDIA optimizers."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import constant_op as cop
+from tensorflow.python.framework import dtypes
+from tensorflow.core.framework import attr_value_pb2
+
+
+def numerical_hint(op, max_value, min_value):
+  """Add a const numerical hint node"""
```

Comment:
Could you describe in the comment what exactly this function and the arguments `max_value` and `min_value` do?